src/Google/GlobalSiteTag.php

```diff
@@ -217,16 +217,24 @@ function () use ( $gtag_events ) {
 	 */
 	public function activate_global_site_tag( string $ads_conversion_id ) {
 		if ( $this->gtag_js->is_adding_framework() ) {
-			add_filter(
-				'woocommerce_gtag_snippet',
-				function ( $gtag_snippet ) use ( $ads_conversion_id ) {
-					return preg_replace(
-						'~(\s)</script>~',
-						"\tgtag('config', '" . $ads_conversion_id . "', { 'groups': 'GLA', 'send_page_view': false });\n$1</script>",
-						$gtag_snippet
-					);
-				}
-			);
+			if ( version_compare( \WC_GOOGLE_ANALYTICS_INTEGRATION_VERSION, '2.0.0', '>=' ) ) {
```

Comment:
:construction: 
I don't get into this line when I run it with the new GA, as the one above returns `false`.
GLA still expect the old GA settings to be there https://github.com/woocommerce/google-listings-and-ads/blob/3a423a2d5cfca64486c74cf41905d01db1cc4c52/src/Proxies/GoogleGtagJs.php#L24:L33

The latest GA does not set `ga_gtag_enabled`

Reply:
I've made a fix in https://github.com/woocommerce/google-listings-and-ads/compare/update/global-site-tag-for-analytics-version-2...update/global-site-tag-for-analytics-version-2-enabled?expand=1